.travis.yml

```diff
@@ -1,16 +1,9 @@
-sudo: false
+sudo: required
```

Comment:
I need to understand why sudo. It makes CI slower.

Reply:
Good question, I spent some time to figure it out too.
Current configuration is also using VM instead of containers, but it looks like travis set it implicitly when it finds "sudo" in the commands, [this line for example](https://github.com/aeternity/epoch/pull/362/files#diff-354f30a63fb0907d4ad57269548329e3L11).

You can check if travis is using VM or container by checking the "Worker Information" in [some of the builds](https://travis-ci.org/aeternity/epoch/jobs/302534493).

```
Worker information
hostname: 80a7a052-5d60-45d0-81f3-5bcc0761362a@1.production-5-worker-org-b-2-gce
version: v3.3.1 https://github.com/travis-ci/worker/tree/1153e5010ef9863bf96264b75fba506109633460
instance: travis-job-629e5d66-b179-435d-a315-78ff29a9db87 travis-ci-amethyst-trusty-1503972832 (via amqp)
startup: 21.383891764s
```

Looking the hostname (GCE) and startup time, it's clear it is using VM not container.
[This documents the expected startup time](https://docs.travis-ci.com/user/reference/overview/#Virtualization-environments).

So I just make it explicit because I moved the scripting in the `ci` script, thus travis no more find this implicit requirement.

While I agree that boot time is much faster for containers, it's not always true that the build summary time would be faster. We can discuss and test this in future, and switch to best option (if possible).

Reply:
Good point. Something to consider in the future. Thanks for the explanation.